src/Foundation/NSUrlSessionConfiguration.cs

```diff
@@ -64,5 +65,33 @@ public static NSUrlSessionConfiguration CreateBackgroundSessionConfiguration (st
 			return config;
 		}
 
+#if NET
+		[SupportedOSPlatform ("ios17.0")]
+		[SupportedOSPlatform ("macos14.0")]
+		[SupportedOSPlatform ("maccatalyst17.0")]
+		[SupportedOSPlatform ("tvos17.0")]
+#else
+		[TV (17,0), Watch (10,0), iOS (17,0), Mac (14,0), MacCatalyst (17,0)]
+#endif
+		public NWProxyConfig[] ProxyConfigurations { 
+			get {
+				var proxyConfigurations = _ProxyConfigurations;
+				if (proxyConfigurations.Count == 0)
+					return Array.Empty<NWProxyConfig> ();
+				var result = new NWProxyConfig [proxyConfigurations.Count];
+				for (nuint i = 0; i < proxyConfigurations.Count; i++) {
+					result [i] = new NWProxyConfig (proxyConfigurations.ValueAt(i), owns: false);
+				}
+				return result;
```

Comment:
If you bind _ProxyConfigurations as IntPtr, something like this should work:

```suggestion
				return NSArray.ArrayFromHandleFunc (_ProxyConfigurations, handle => new NWProxyConfig (handle, owns: false));
```